core/src/main/java/com/datastax/oss/driver/internal/core/adminrequest/AdminRow.java

```diff
@@ -94,6 +94,15 @@ public boolean contains(String columnName) {
     return columnSpecs.containsKey(columnName);
   }
 
+  public boolean isNotEmpty(String columnName) {
+    if (contains(columnName)) {
+      int index = columnSpecs.get(columnName).index;
+      ByteBuffer bb = data.get(index);
+      return bb != null && bb.remaining() > 0;
```

Comment:
This should cover empty collections too. It will also catch empty strings, but I think it's fine.